src/backend/distributed/metadata/node_metadata.c

```diff
@@ -496,16 +496,6 @@ citus_disable_node(PG_FUNCTION_ARGS)
 								 workerNode->workerName,
 								 nodePort)));
 		}
-
```

Comment:
Can we add a comment here to explain having placement metadata related to disabled node will stay till the next remove_node or active_node?